README.md

```diff
@@ -35,6 +35,11 @@ desired configuration.
 
 ## Installing and running
 
+UsbDk can be installed via Winget:
```

Comment:
```suggestion
UsbDk can be installed via Winget on desktop Windows:
```

Reply:
Shouldn't this be obvious? UsbDk and Winget is Windows-only.

Reply:
The idea is to emphasize the fact that this way is for desktop versions only, not server